src/tasks/level4.ts

```diff
@@ -82,7 +154,7 @@ export const BatQuest: Quest = {
           return stenchPlanner.outfitFor(1, {
             equip: $items`industrial fire extinguisher`,
           });
-        else return stenchPlanner.outfitFor(1, { modifier: "item" });
+        else return stenchPlanner.outfitFor(1, { modifier: "item", equip: $items`bat wings` });
```

Comment:
We don't need lines 157 / 208 / 234 / 266 now that we have bat-wings-specific tasks, right?